Src/Notion.Client/Models/Page/RetrievedPage.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Notion.Client
 {
-    public class Page
+    public class RetrievedPage
```

Comment:
@asevos I would like to keep the name of class as `Page` reason being retrieving a page is more of action but the Page is an object.

Reply:
Ok, renamed it back to `Page`